common/api/imodeljs-common.api.md

```diff
@@ -3742,7 +3735,7 @@ export abstract class IModel implements IModelProps {
     get changeSetId(): string | undefined;
     // @internal (undocumented)
     protected _changeSetId: string | undefined;
-    get contextId(): GuidString | undefined;
+    get contextId(): string | undefined;

```

Comment:
Why change from GuidString --> string?

Reply:
No clue. It's GuidString in the real code, just extract-api reduced it to string for some reason.